src/WebUI/Utils.ts

```diff
@@ -50,7 +50,9 @@ export class Utils {
         let tags: string[] = [];
         if (items) {
             Object.keys(items).forEach((key: string) => {
-                tags.push(format("{0}={1}", key, items[key]));
+                // trim single/double quotes
+                const keyValue = (items[key] || "").trim().replace(/^("|')+/, "").replace(/("|')+$/, "");

```

Comment:
I don't understand what the two replace statements are doing. The second parameter in both replace functions is the same.

Reply:
replace can use regex, in the first replace, we are replacing a quote or double quote at the start of the string ('^' indicates start) with a empty value. same with end of the string also.